runtime/gc_base/FinalizerSupport.cpp

```diff
@@ -592,9 +603,17 @@ static int J9THREAD_PROC FinalizeSlaveThread(void *arg)
 			}
 #endif /* J9VM_GC_DYNAMIC_CLASS_UNLOADING */
 
-			// processing will release/acquire VM access
+			/* processing will release/acquire VM access */
 			process(env, finalizeJob, j9VMInternalsClass, runFinalizeMID, referenceEnqueueImplMID);
 
+			if (vm->processReferenceMonitor && (0 != vm->processReferenceActive)) {
```

Comment:
Same comment for the null check on `vm->processReferenceMonitor`

Reply:
updated